src/components/ChecFilterBar/FilterPanel.vue

```diff
@@ -277,7 +277,13 @@ export default {
   }
 
   &__actions {
-    @apply pt-6 pb-2 flex justify-between;
+    @apply pt-6 pb-2 flex justify-end;
+
+    > button {
```

Comment:
```suggestion
    .button {
```

Does this work? It's less specific. You could also add `space-x-2` or something to the parent container